orchagent/portsorch.cpp

```diff
@@ -19,6 +19,80 @@ extern sai_hostif_api_t* sai_hostif_api;
 #define VLAN_PREFIX         "Vlan"
 #define DEFAULT_VLAN_ID     1
 
+namespace swss {
```

Comment:
Port class is member of swss namespace. As we've agreed in past, this code was placed in this file.